ztp/tools/pgt2acm/test/schema.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
Maybe rename the file to something ~ `ptpconfig_schema.json`?

Reply:
Fixed. I deleted the schema.json.